docs/crypto.md

```diff
@@ -36,6 +36,20 @@ It takes one of the values for its first param:
 - `dsa`: Generate a DSA key (L2048N256)
 - `rsa`: Generate an RSA 4096 key
 
+## getCA
+
+The `getCA` function allows customing the Root CA that used by `genSignedCert`.
```

Comment:
s/customing/customizing/

Reply:
Also, the description artificially constrains the use. See comment above about the name.